doc/_templates/layout.html

```diff
@@ -1,5 +1,6 @@
 {% extends "!layout.html" %}
 
 {%- block extrahead %}
-    <script type="text/javascript" src="http://ayni.ceph.com/public/js/ceph.js"></script>
+   <!--  <script type="text/javascript"
```

Comment:
could just remove file. in other words, to revert 35663fa55ac1579a3b0c8b67028a3a8dfea87b48. but let's wait for @alfredodeza 's call.